app/models/credence_game_response.rb

```diff
@@ -0,0 +1,70 @@
+class CredenceGameResponse < ActiveRecord::Base
+  belongs_to :credence_game
+  belongs_to :credence_question
+  belongs_to :first_answer, class_name: 'CredenceAnswer'
+  belongs_to :second_answer, class_name: 'CredenceAnswer'
+
+  def self.pick_random
+    enabled_questions = CredenceQuestion.where(enabled: true)
+    num_enabled = enabled_questions.count
```

Comment:
num_enabled reads like: "The option num is enabled by being set to true"

I would avoid putting this in a temporary variable and instead just do the following below:
```
    question = enabled_questions.first(offset: rand(enabled_questions.count))
     while !question.enabled || question.weight < rand
       question = enabled_questions.first(offset: rand(enabled_questions.count))
     end
```

All you are accomplishing by putting this in a temp is to make the code harder to read.